pkg/cmd/experimental/syncgroups/groupsyncer.go

```diff
@@ -119,7 +143,7 @@ func (s *LDAPGroupSyncer) findGroup(ldapGroupUID string) (*ouserapi.Group, error
 		if s.SyncExisting {
```

Comment:
the line above this should be checking if the `IsNotFound(err)`, not just `err != nil`